lib/bugsnag/api/client/collaborators.rb

```diff
@@ -0,0 +1,87 @@
+module Bugsnag
+  module Api
+    class Client
+
+      # Methods for the Organizations Collaborators API
+      #
+      # @see http://docs.bugsnagapiv2.apiary.io/#reference/organizations/collaborators
+      module Collaborators
+        # View a Collaborator
+        #
+        # @return [Sawyer::Resource] Collaborator
+        # @see http://docs.bugsnagapiv2.apiary.io/#reference/organizations/collaborators/view-a-collaborator
+        def collaborator(org_id, collaborator_id, options = {})
+            get "organizations/#{org_id}/collaborators/#{collaborator_id}", options
```

Comment:
The methods in this file are differently indented from the class and module declarations